XamlStyler.Package/StylerPackage.cs

```diff
@@ -201,6 +209,32 @@ private void Execute(Document document)
             }
         }
 
+        private string GetConfigPathForItem(string path)
+        {
+            try
+            {
+                if (string.IsNullOrWhiteSpace(path))
+                {
+                    return null;
+                }
+
+                while ((path = Path.GetDirectoryName(path)) != null)
```

Comment:
I suppose it depends on whether the spirit behind this feature is to support teams distributing settings in a repo or for a developer to have finer-grain control over their own settings. Both seem valid (and do not necessarily exclude each other), but IMO, the team scenario is most compelling. In that case, perhaps it is not root of solution, but root of repo, like .gitignore? I'll let others chime in.